docs/install/install.md

```diff
@@ -0,0 +1,29 @@
+
+
+
+
+# Installing Half
+
+The library in its most recent version can be obtained from here, see the [Release Notes](changelog.html) for further information:
+
+<ul class="tablist"><li>[Download half 1.12.0 (.zip)](http://sourceforge.net/projects/half/files/latest/download)</li></ul>
```

Comment:
This formatting looks messed up.
![image](https://github.com/ROCm/half/assets/164940351/97a476dc-078a-4b98-8fe6-6ad7d44acc98)